packages/vendure-plugin-customer-managed-groups/src/customer-managed-groups.service.ts

```diff
@@ -446,6 +479,22 @@ export class CustomerManagedGroupsService {
       ...(input.emailAddress ? { emailAddress: input.emailAddress } : []),
     };
     await this.customerService.update(ctx, updateUserData);
+    if (input.addresses && input.addresses.length) {
+      for (let addressInput of input.addresses) {
+        if (addressInput && (addressInput as any).id) {
```

Comment:
```suggestion
        if (addressInput?.id) {
```
Do we need to assume `as any`? It should be a generated type right?